src/features/bank-request-info/bank-services.ts

```diff
@@ -14,6 +14,19 @@ export const services: Service[] = [
       "Diamond (Price on Jewelry Spreadsheet, _North Freeport Bank_)",
     ],
   },
+  {
+    title: "OT Hammer Event Scheduling",
+    icon: Icon.Sold,
+    requestFormats: ["OT Hammer Event", "OT Hammer Event: Date, Time (include timezone)"],
+    bullets: [
+      "Requires more people to facilitate, so this is a request to schedule an event",
+      "You bring a Jade and 1750pp (for Alliance members)",
+      "We provide CHA gear, Tash and Malo",
+      "No IOUs: Payment made in NFP or on-site",
+      "Default time: Thursday at noon EST",
+      "Non-alliance members: cost is a Jade and 1950pp (and we reserve the right to refuse service)",
```

Comment:
Since this isn't visible to non alliance players, I'm not sure we need to include it. Doesn't hurt tho I guess.

Reply:
Ben asked for it; my sense is it is at least a note to someone in alliance who knows someone outside of alliance who wants 🤷